src/DataStructures/Tensor/EagerMath/OuterProduct.hpp

```diff
@@ -0,0 +1,67 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace OuterProduct_detail {
+template <typename DataType, typename SymmLhs, typename IndicesLhs,
+          typename SymmRhs, typename IndicesRhs>
+using ResultTensor = Tensor<
```

Comment:
[optional] I need this metafunction for IMEX.  You can make it public here or I can do it when I merge that code.  Up to you.

Reply:
Sure, added a fixup